packages/styles/scss/components/card-group/_card-group.scss

```diff
@@ -80,10 +80,8 @@
       }
 
       .#{$prefix}--card__content {
-        display: grid;
-        grid-row: span 10;
-        grid-template-rows: subgrid;
-        row-gap: 0;
+        display: flex;
+        min-block-size: 10rem;
```

Comment:
Hum, this won't work, as it breaks the [fixes for sameheight](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/12063).

![image](https://github.com/user-attachments/assets/74f70b71-55a6-44b9-b92e-2d9bdf074f19)

Compare to: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components/?path=/story/components-card-group--default